tests/test_formats.py

```diff
@@ -35,7 +35,7 @@ def test_icl_format_with_demonstrations(self):
         self.assertEqual(result, target)
 
     def test_icl_format_with_demonstrations_and_instruction_after_demos(self):
```

Comment:
once changed to operator it will be better if icl format will be tested with the operator testing function

Reply:
Done, Comparing both results.